packages/compat/src/__tests__/signed-versioned-transaction-test.ts

```diff
@@ -0,0 +1,142 @@
+import { Keypair, TransactionMessage, VersionedTransaction } from '@solana/web3.js';
+
+import {
+    assertIsSignedVersionedTransaction,
+    isSignedVersionedTransaction,
+    signedVersionedTransaction,
+} from '../signed-versioned-transaction';
+
+describe('signed-versioned-transactions', () => {
+    const payerKey = Keypair.generate().publicKey;
+    const blockhashString = '4JgzET199353yzPPZ7QUAFa3FggaLmkeMczKwqW4VqTT';
+    const signature = new Uint8Array(Array(64).fill(1));
```

Comment:
Small nit: since this puppy is mutable, you might consider re-initializing it on every pass.
```suggestion
    let signature: Uint8Array;
    beforeEach(() => {
        signature = new Uint8Array(Array(64).fill(1));
    });
```

Reply:
Same with `payerKey`.

Reply:
Resolving since these tests are removed